resources/js/group.js

```diff
@@ -1,10 +1,11 @@
 export default class Group {
 
-    constructor(name, title, fields, field, key) {
+    constructor(name, title, fields, field, key, collapsed) {
         this.name = name;
         this.title = title;
         this.fields = fields;
         this.key = key || this.getTemporaryUniqueKey(field.attribute);
+        this.collapsed = (collapsed === undefined) ? true : collapsed,
```

Comment:
You can indeed make this better by using [default parameters](https://css-tricks.com/using-default-parameters-es6/#default-parameters-in-es6).

Reply:
Added